test/test_helper.rb

```diff
@@ -3,6 +3,7 @@
 require "rails"
 require "rails/test_help"
 require "byebug"
+require "ostruct"
```

Comment:
Without this tests were failing with 

```
Error:
CompilerTest#test_freshness_on_compile_success:
NameError: uninitialized constant CompilerTest::OpenStruct
```

e.g. on https://github.com/rails/webpacker/actions/runs/9956692322/job/27507151674#step:4:83